contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -75,8 +75,8 @@ const (
 	defaultReconcileCooldown = 15 * time.Second
 	defaultFrameworkName     = "Kubernetes"
 
-	executorCPUs = mresource.CPUShares(0.25) // initial CPU allocated for executor
-	executorMem  = mresource.MegaBytes(64.0) // initial memory allocated for executor
+	ExecutorCPUs = mresource.CPUShares(0.01) // initial CPU allocated for executor, intentionally set to the minimum allow by Mesos: 0.01
```

Comment:
is this really enough for production use? with large numbers of pods running on a host and given that kubelet is also running cadvisor and constantly pinging docker containers for status .. a value of `0.01` seems quite insufficient. I suspect that the older value `0.25` would also prove insufficient at certain scales.

if this was done for testing, then I think we need to do a couple of things:
- document that fact here (and also, perhaps, in the known issues doc?)
- allow this default value to be overridden at runtime so that people doing real work w/ this framework don't suffer

if this was not done for testing then I recommend leaving value alone.